### ISSUE TITLE
Out of order tags are confusing Dependabot

### ISSUE THREAD

**Issue by bwestover** [okteto/pipeline#23]
We use Dependabot to manage our GitHub Actions dependencies ([docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates)). 

The issue we're running into is that this repo has "newer" tags (v4.2 - v4.6) than the actual latest as of this writing (2.5.2). Those tags point to older code, and so Dependabot is trying to "upgrade" us to an older version. Other dependency managers may have a similar issue, I'm not sure.

Would you consider removing these out of order tags?

@rberrelleza @irespaldiza

**Comment by rberrelleza** [comment]
@bwestover yes, that's a good idea. I think we should remove the old tags, since we are now following a different version scheme. Are you only affected by pipeline action?

**Comment by bwestover** [comment]
> Are you only affected by pipeline action?

We also saw this in `okteto/login`, however I believe that one is deprecated anyway in favor of `okteto/context`. The other actions we use (`okteto/destroy-preview` and `okteto/deploy-preview`) didn't seem to have the out of order tags, so were working correctly with Dependabot.

Thanks!

**Comment by maroshii** [comment]
Thanks for reporting @bwestover! This should be fixed now. All `v4.x` tags have been deleted.

**Comment by bwestover** [comment]
:sparkles: Thank you :bow:
